packages/core/admin/admin/src/hooks/useFetchMarketplaceProviders/index.js

```diff
@@ -2,10 +2,10 @@ import { useQuery } from 'react-query';
 import { useNotification } from '@strapi/helper-plugin';
 import { fetchMarketplacePlugins } from './utils/api';
 
-const useFetchMarketplaceProviders = (notifyLoad) => {
+const useFetchMarketplaceProviders = (notifyLoad, sort) => {
```

Comment:
It would be more convenient to ask for a params object instead of just sort here, for when we add collections and categories